Meta/Web Foundations team responsibilities.md

```diff
@@ -0,0 +1,67 @@
+
+## Language and platform experts
+
+While we expect all Web and UX Developers at Shopify to have a solid understanding of the web platform and associated technologies, the Web Foundation team is meant to serve as a consistent group of experts in these technologies that can assist other teams in addressing the Web, irrespective of the expertise they have on their own team. The Web Foundation team promises to maintain a modern and thorough understanding of the following:
+
+* JavaScript (and variants in common use at Shopify, such as TypeScript)
+* Sass and CSS
+* HTML
+* DOM and other browser APIs
+* Node
+* GraphQL
+* Frameworks and libraries in common use at Shopify, such as React and Koa
+
+Given this broad expertise in the platform and technologies of the Web, the Web Foundation team act as the stewards for our code style guides and the shared linting/ formatting configurations derived from these style guides. The Web Foundation team will, when necessary, break deadlocks in bike shedding around the stylistic use of the above technologies.
+
+Additionally, the Web Foundation team should be at least a "keep informed" stakeholder on any project that involves a key extension or change to our typical process of developing for the Web, and for any project that involves significant and novel use of Web technologies not done elsewhere in Shopify.
+
+## Stewards of common tools, libraries, and best practices
+
+At Shopify, we do not believe that every project should have completely free-reign in deciding their product’s tech stack. We expect projects to use our common developer tools, deploy systems, and production environments. We also make "picks" at higher levels in the stack that we believe are better served by consistency than developer choice, such as our preference for Rails as an application framework, or our preference for MySQL as the primary data store. These choices are often made in an effort to provide a minimum level of quality from all the products we create.
+
+The Web Foundation team applies this same principle of having sensible "picks" to developer tools, application libraries, and best practices specific to the Web platform. However, we are mindful of not creating "thin clients" — that is, leaving no decisions to application teams about what their application should look like.
+
+It is useful to consider the separation of powers between federal and provincial governments, as this is a similar model to how the Web Foundation team considers topics in which to involve themselves. In Canada, the federal government delegates the responsibility for overseeing schools, hospitals, and property rights to provincial governments, while retaining powers that they are best positioned to address, such as national defence and currency. Similarly, the Web Foundations team is responsible for guiding decisions on topics that are important for all applications, but delegates most of the finer points to application teams.
```

Comment:
Not sure how useful this paragraph is.

Reply:
I agree here. I personally like the analogy but fear others may not.

Reply:
:sad_panda: